content/en/install.md

```diff
@@ -3,101 +3,148 @@ title: Installation
-  Windows, and Mac.
+[venv-guide]: https://packaging.python.org/en/latest/tutorials/installing-packages/#creating-virtual-environments
+[SciPy from PyPI]: https://pypi.org/project/scipy/
+[`pip`]: https://pip.pypa.io/en/stable/getting-started/
 
-<a name="pip-install"></a>
+Alternatively, you can install [SciPy from conda-forge] using a [Conda]-based
+environment manager. You can install Python itself, as well as packages in
+languages other than Python, as Conda packages.
 
-# Installing with Pip
+[Miniforge] is the recommended way to install `conda` and [`mamba`],
+two Conda-based environment managers. You can then install SciPy as follows:
 
-You can install SciPy from PyPI with `pip`:
+    conda install scipy # or
+    mamba install scipy
 
-    python -m pip install scipy
+[SciPy from conda-forge]: https://anaconda.org/conda-forge/scipy
+[Miniforge]: https://conda-forge.org/download/
+[Conda]: https://docs.conda.io/projects/conda/en/latest/index.html
+[`mamba`]: https://mamba.readthedocs.io/en/latest/
+
+<a name="project-based"></a>
+
+## Project-based workflows
+
+In project-based workflows, a project is a directory containing a manifest
+file describing the project, a lock-file describing the exact dependencies
+of the project, and the project's (potentially multiple) environments.
+These workflows provide reproducibility benefits over environment-based workflows.
+
+You can add [SciPy from PyPI] to a [`uv`] project:
+
+    uv add scipy
+
+You can also install Python itself with `uv`.
+
+[`uv`]: https://docs.astral.sh/uv/
+
+If you work with non-Python packages, you may prefer to install SciPy as
+a Conda package, so that you can use the same workflow for packages which
+are not available on PyPI.
 
-<a name="conda-install"></a>
+You can add [SciPy from conda-forge] to a [`pixi`] project:
 
-# Installing via Conda
+    pixi add scipy
 
-You can install SciPy from the `defaults` or `conda-forge` channels with `conda`:
+Pixi can also install packages from PyPI, using `uv` under the hood.
 
-    conda install scipy
+[`pixi`]: https://pixi.sh/latest/
 
-<a name="package_manager"></a>
+<a name="system-package-managers"></a>
 
-# Install system-wide via a package manager
+## Installing system-wide via a system package manager
 
 System package managers can install the most common Python packages.
 They install packages for the entire computer, often use older versions,
-and don't have as many available versions.
+and don't have as many available versions. They are not the recommended
+installation method, but they are the simplest.
 
-## Ubuntu and Debian
+### Ubuntu and Debian
 
 Using `apt-get`:
 
     sudo apt-get install python3-scipy
 
-## Fedora
+### Fedora
 
 Using `dnf`:
 
     sudo dnf install python3-scipy
 
-## macOS
+### macOS
 
 macOS doesn't have a preinstalled package manager, but you can install
 [Homebrew](https://brew.sh/) and use it to install SciPy (and Python itself):
 
     brew install scipy
```

Comment:
`brew` is falling more and more out of favour as it has a horrible permissions model: Only one user can use it on one system OOTB. I'd recommend MacPorts if we want something Mac-specific.

Reply:
I did not hear MacPort being mentioned in years. Both around OSS and professionally. Do you have any supporting material to back that statement up? 😅

Reply:
About the multi-user issue with Homebrew:

> If you need to run Homebrew in a multi-user environment, consider creating a separate user account specifically for use of Homebrew.

From [the docs](https://docs.brew.sh/FAQ#why-does-homebrew-say-sudo-is-bad).

A lot of workarounds for per-user installs are unsupported: [1](https://docs.brew.sh/Installation#untar-anywhere-unsupported) [2](https://docs.brew.sh/Installation#multiple-installations-unsupported).

I don't use MacPorts personally; mostly `pixi` these days; but Homebrew is a potential footgun for the uninitiated.

Reply:
MacPorts is still a thing, but we should keep Homebrew. It's the most popular package manager on macOS, and the limitation of no per-user installs on a shared machine is pretty niche, I haven't seen that come up as an issue before. We can add MacPorts of course. Both aren't recommended either way I'd say, since they're similar to system package managers. Users are better off knowing how to create environments (whether venvs or conda envs).

Reply:
> the limitation of no per-user installs on a shared machine is pretty niche

That's one limitation -- another is that the global installation cannot be ergonomically used by multiple users. The only supported way to do that is to imitate another user; as the permissions are tied to a user, not a group.

Reply:
This is related to the "projects" discussion, but Homebrew has recently made their Python installation considerably less friendly to beginners, by implementing the "virtualenv-only pip" PEP - and in particular, strongly deprecating `--user` installs - see discussion at https://github.com/orgs/Homebrew/discussions/3404 .